i18n/testdata/two_questions.en.po

```diff
@@ -0,0 +1,62 @@
+# Testing
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2020-03-25 13:57+0000\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+
+#: 615b8a0f-588c-4d20-a05f-363b0b4ce6f4/d2a4052a-3fa9-4608-ab3e-5b9631440447/text:0
+msgid "@(TITLE(results.favorite_color.category_localized)) it is! What is your favorite soda? (pepsi/coke)"
+msgstr ""
+
+#: 615b8a0f-588c-4d20-a05f-363b0b4ce6f4/c70fe86c-9aac-4cc2-a5cb-d35cbe3fed6e/name:0,615b8a0f-588c-4d20-a05f-363b0b4ce6f4/e97cd6d5-3354-4dbd-85bc-6c1f87849eec/quick_replies:1
+msgid "Blue"
+msgstr ""
+
+#: 615b8a0f-588c-4d20-a05f-363b0b4ce6f4/c7bca181-0cb3-4ec6-8555-f7e5644238ad/name:0
+msgid "Coke"
+msgstr ""
+
+#: 615b8a0f-588c-4d20-a05f-363b0b4ce6f4/0a8467eb-911a-41db-8101-ccf415c48e6a/text:0
+msgid "Great, you are done and like @results.soda.value! Webhook status was @results.webhook.value"
+msgstr ""
+
+#: 615b8a0f-588c-4d20-a05f-363b0b4ce6f4/e97cd6d5-3354-4dbd-85bc-6c1f87849eec/text:0
+msgid "Hi @contact.name! What is your favorite color? (red/blue) Your number is @(format_urn(contact.urn))"
+msgstr ""
+
+#: 615b8a0f-588c-4d20-a05f-363b0b4ce6f4/1024833c-91aa-4873-a3b5-3bac1ef55812/name:0
+msgid "No Response"
+msgstr ""
+
+#: 615b8a0f-588c-4d20-a05f-363b0b4ce6f4/5ce6c69a-fdfe-4594-ab71-26be534d31c3/name:0,615b8a0f-588c-4d20-a05f-363b0b4ce6f4/78ae8f05-f92e-43b2-a886-406eaea1b8e0/name:0
+msgid "Other"
+msgstr ""
+
+#: 615b8a0f-588c-4d20-a05f-363b0b4ce6f4/2ab9b033-77a8-4e56-a558-b568c00c9492/name:0
+msgid "Pepsi"
+msgstr ""
+
+#: 615b8a0f-588c-4d20-a05f-363b0b4ce6f4/598ae7a5-2f81-48f1-afac-595262514aa1/name:0,615b8a0f-588c-4d20-a05f-363b0b4ce6f4/e97cd6d5-3354-4dbd-85bc-6c1f87849eec/quick_replies:0
```

Comment:
string appears in two places - a category name and a quick reply

Reply:
Any reason not to use two lines here instead of a comma? (notice Django uses separate lines). Also wonder if it may be worth annotating with the flow name just for anybody who is peeping the file? (we would still use UUID but just for reference sake)

```
#: temba/api/v2/views.py:278 temba/orgs/models.py:2213
#: templates/orgs/org_manage_accounts.haml:220
msgid "Administrator"
msgstr ""
```

Reply:
(just noticed the django version is a bit of a mix of newlines vs same line actually)

Reply:
And just to be clear on the direction we decided, if "Red" appeared twice but was translated separately in the two places, then.. we add a `msgctxt`? Or we use the latest?

Reply:
Didn't know you could put them on separate lines.. guessing that's different versions of `xgettext` doing that..  certainly easier to read on separate lines so I can update that if poedit can handle it

Reply:
And re second question, we did decide that we include context where it's needed to separate differing translations of the same base - and that in theory the translator could fix that themselves if they decided it was unintentional. Making a test file for that scenario currently.

Reply:
Switched to using flow name as part of the reference - but URL encoding it to avoid problems with spaces. I think it's fine to use name over UUID here - these aren't real links.. you can't click on them to jump into the source code.. so they're just tips for the translator.

Reply:
Just tried and POEdit does preserve dual comments in generated translation files.